website/src/components/Macrobench/Macrobench.jsx

```diff
@@ -26,8 +26,8 @@ const Macrobench = ({data, gitRefLeft, gitRefRight}) => {
             <table>
                 <thead>
                     <tr>
-                        <th><h4>{gitRefLeft}</h4></th>
-                        <th><h4>{gitRefRight}</h4></th>
+                        <th><h4>{gitRefLeft.slice(0, 20)}</h4></th>
+                        <th><h4>{gitRefRight.slice(0, 20)}</h4></th>
```

Comment:
Mhm... I don't think it is fair to trim the git ref here. I think the caller of the `Macrobench` component should be the one responsible for trimming it.